CreeDictionary/API/migrations/.gitignore

```diff
@@ -0,0 +1 @@
+/*.py
```

Comment:
see the new readme.md for details

Reply:
As a formatting note, I would present the stem more prominently, similar to what was done previously with the lemma in the popup (so **bolded**).  The lemma/dictionary entry is visible already anyhow.